.github/workflows/ci-17.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Test
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+
+    name: Build and Test (17)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
```

Comment:
Do we want to test both Java 11 and 17 here?
If so, we should rename the file

Reply:
We build with 11 but test with 17 here, so really this is a 17 test.